### PR TITLE
refactor: refactor cdk

### PR DESCRIPTION
**Pull request by xdlbdy**
**Changes:**
-

### CODE REVIEW

File: core/src/main/java/com/linkall/vance/core/VanceApplication.java [hunk 1]
```diff
@@ -16,65 +16,85 @@
...
+import com.linkall.vance.config.ConfigUtil;
+import com.linkall.vance.config.SinkConfig;
+import com.linkall.vance.config.SourceConfig;
+import com.linkall.vance.core.runtime.ConnectorWorker;
+import com.linkall.vance.core.runtime.http.SinkWorker;
+import com.linkall.vance.core.runtime.http.SourceWorker;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class VanceApplication {
```
Comment (wenfengwang) [inline core/src/main/java/com/linkall/vance/core/VanceApplication.java:29]: rename to `VanceApplication`?

File: core/src/main/java/com/linkall/vance/store/KVStore.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.linkall.vance.store;
+
+public interface KVStore {
+
+    void put(String key, byte[] value) throws Exception;
```
Comment (wenfengwang) [inline core/src/main/java/com/linkall/vance/store/KVStore.java:5]: set?

**Review by wenfengwang — CHANGES_REQUESTED** [review]

**Comment by wenfengwang** [comment]
Here are two need to disscuss:
1. change `core` to `runtime` in order to keep same with cdk-go
2. change package to `com.linkall.cdk`

File: core/src/main/java/com/linkall/vance/core/VanceApplication.java (hunk 1 above)
Comment (xdlbdy) [reply core/src/main/java/com/linkall/vance/core/VanceApplication.java:29]: do you mean Application?

**Review by wenfengwang — APPROVED** [review]
LGTM

**Pull request merged**